<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1411</width>
    <height>1046</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,0,9">
      <item>
       <widget class="QDockWidget" name="dockWidget">
        <property name="floating">
         <bool>false</bool>
        </property>
        <property name="features">
         <set>QDockWidget::DockWidgetMovable</set>
        </property>
        <property name="allowedAreas">
         <set>Qt::AllDockWidgetAreas</set>
        </property>
        <widget class="QWidget" name="dockWidgetContents">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeView" name="treeView"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="searchEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="searchBtn">
                <property name="text">
                 <string>搜索</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDockWidget" name="dockWidget_2">
        <property name="floating">
         <bool>false</bool>
        </property>
        <property name="features">
         <set>QDockWidget::AllDockWidgetFeatures</set>
        </property>
        <property name="allowedAreas">
         <set>Qt::AllDockWidgetAreas</set>
        </property>
        <widget class="QWidget" name="dockWidgetContents_2">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTabWidget" name="tabWidget">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <property name="tabsClosable">
               <bool>false</bool>
              </property>
              <property name="movable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="tab_2">
               <attribute name="title">
                <string>数据</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                 <widget class="QTabWidget" name="tabWidget_2">
                  <property name="tabPosition">
                   <enum>QTabWidget::South</enum>
                  </property>
                  <property name="tabShape">
                   <enum>QTabWidget::Rounded</enum>
                  </property>
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <property name="documentMode">
                   <bool>true</bool>
                  </property>
                  <property name="tabsClosable">
                   <bool>true</bool>
                  </property>
                  <property name="movable">
                   <bool>true</bool>
                  </property>
                  <property name="tabBarAutoHide">
                   <bool>false</bool>
                  </property>
                  <widget class="QWidget" name="tab_3">
                   <attribute name="title">
                    <string>Tab 1</string>
                   </attribute>
                   <layout class="QGridLayout" name="gridLayout_2">
                    <item row="0" column="1">
                     <widget class="QTableWidget" name="tableWidget_2"/>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="tab_4">
                   <attribute name="title">
                    <string>Tab 2</string>
                   </attribute>
                   <layout class="QGridLayout" name="gridLayout_4">
                    <item row="0" column="0">
                     <widget class="QTableWidget" name="tableWidget"/>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab">
               <attribute name="title">
                <string>图片</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout">
                <item row="1" column="0">
                 <widget class="MatplotlibWidget" name="MatWidget" native="true"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QSlider" name="horizontalSlider">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="invertedAppearance">
                 <bool>false</bool>
                </property>
                <property name="invertedControls">
                 <bool>false</bool>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::NoTicks</enum>
                </property>
                <property name="tickInterval">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="drawBtn">
                  <property name="text">
                   <string>绘制</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox"/>
                </item>
                <item>
                 <widget class="QDateTimeEdit" name="dateTimeEdit">
                  <property name="calendarPopup">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1411</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="AddBattle"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewProject"/>
   <addaction name="actionOpenProject"/>
   <addaction name="actionSaveProject"/>
   <addaction name="separator"/>
   <addaction name="actionLookData"/>
   <addaction name="actionLookPicture"/>
   <addaction name="separator"/>
   <addaction name="actionSectting"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="AddBattle">
   <property name="text">
    <string>添加战斗</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../demo/icon/res.qrc">
     <normaloff>:/icon/jurassic_add-project.png</normaloff>:/icon/jurassic_add-project.png</iconset>
   </property>
   <property name="text">
    <string>新建项目</string>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset resource="../../demo/icon/res.qrc">
     <normaloff>:/icon/jurassic_open.png</normaloff>:/icon/jurassic_open.png</iconset>
   </property>
   <property name="text">
    <string>打开项目</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="icon">
    <iconset resource="../../demo/icon/res.qrc">
     <normaloff>:/icon/保存_操作_jurassic.png</normaloff>:/icon/保存_操作_jurassic.png</iconset>
   </property>
   <property name="text">
    <string>保存项目</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionLookPicture">
   <property name="icon">
    <iconset resource="../../demo/icon/res.qrc">
     <normaloff>:/icon/查看图片 (1).png</normaloff>:/icon/查看图片 (1).png</iconset>
   </property>
   <property name="text">
    <string>查看图片</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionLookData">
   <property name="icon">
    <iconset resource="../../demo/icon/res.qrc">
     <normaloff>:/icon/jurassic_preview.png</normaloff>:/icon/jurassic_preview.png</iconset>
   </property>
   <property name="text">
    <string>查看数据</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSectting">
   <property name="icon">
    <iconset resource="../../demo/icon/res.qrc">
     <normaloff>:/icon/管理设置_操作_jurassic.png</normaloff>:/icon/管理设置_操作_jurassic.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../../demo/icon/res.qrc">
     <normaloff>:/icon/19-帮助中心.png</normaloff>:/icon/19-帮助中心.png</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../demo/icon/res.qrc"/>
 </resources>
 <connections/>
</ui>
